src/NUnitFramework/tests/Internal/TestExecutionContextTests.cs

```diff
@@ -256,6 +256,13 @@ public void TestCanAccessItsOwnProperties()
             Assert.That(TestExecutionContext.CurrentContext.CurrentTest.Properties.Get("Answer"), Is.EqualTo(42));
         }
 
+        [Test]
```

Comment:
Although permitted, we don't use the extra `[Test]` in our own tests - except where we test that it works, of course.